x11-apps/scripts/scripts-1.0.1-r1.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=3
+EAPI=5
 
 XORG_STATIC="no"
```

Comment:
in `src_install`:
`rm "${ED%/}"/usr/bin/xauth_switch_to_sun-des-1 || die`

also free line before `RDEPEND`